GameData/RP-1/Contracts/Early Space Stations/Early Station Supply.cfg

```diff
@@ -0,0 +1,200 @@
+		durationText = 30 * @/durationMultiplier.ElementAt(@/index)
+	}
+	DATA
+	{
+		type = float 
+		experimentCompletion = @/experimentList.ElementAt(@/index)
+	}
+
+	DATA
+	{
+		type = int
+		index = $RepeatEarlySpaceStation_Count + 0
+	}
+
+	
+	// ************ PARAMETERS ************
+	
+	PARAMETER
+	{
+		name = vesselGroupLaunch
+		type = VesselParameterGroup
+		title = Control a Space Station with the required parameters
+		notes = The station must include at least 1 Docking Port and support for at least 3 Crew
+		define = secondGenerationSpaceStation
+		disableOnStateChange = true
+		
+		PARAMETER
+		{
+			name = HasCapacity
+			type = HasCrewCapacity
+			minCapacity = 3
+			title = Space for at least 3 crew
+			hideChildren = true
+			disableOnStateChange = false
+		}
+		PARAMETER
+		{
+			name = DockingPort
+			type = PartValidation
+			partModule = ModuleDockingNode
+			minCount = 1
+			title = Have at least 1 docking port
+			hideChildren = true
+		}
+		PARAMETER
+		{
+		    name = researchPayloadCapacity //make sure you can hold at least 30 days of research payload
+		    type = HasResourceCapacity
+			
+			resource = ResearchPayload
+			minQuantity = 750
+		}	
+		PARAMETER
+		{
+			name = Orbit
+			type = Orbit
+			minPeA = 250000
+			maxApA = 500000
+			targetBody = HomeWorld()
+			title = Reach orbit with a perigee greater than 250 km and an apogee less than 500 km
+			hideChildren = true
+		}
+	}
+	PARAMETER
+	{
+		name = vesselGroupService
+		type = VesselParameterGroup
+		title = Send any needed supplies, modules, or crew to the station
+		define = serviceVessel
+		completeInSequence = true
+		resetChildrenWhenVesselDestroyed = true
+		
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a Supply Mission
+			hideChildren = true
+			disableOnStateChange = true
+		}
+		PARAMETER
+		{
+			name = DockingPort
+			type = PartValidation
+			partModule = ModuleDockingNode
+			minCount = 1
+			title = Have at least 1 docking port
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = Rendezvous
+			type = Rendezvous
+			vessel = secondGenerationSpaceStation
+			distance = 100
+			title = Rendezvous with the station.
+		}
+		PARAMETER
+		{
+			name = Docking
+			type = Docking
+			vessel = secondGenerationSpaceStation
+			completeInSequence = true
+			title = Dock with the station.
+		}
+
+	}
+	PARAMETER
+	{
+		name = vesselGroupStationPersist
+		type = VesselParameterGroup
+		title = Keep the station in orbit
+		vessel = secondGenerationSpaceStation
+		completeInSequence = true
+		
+		PARAMETER
+		{
+			name = Orbit
+			type = Orbit
+			minPeA = 250000
+			maxApA = 500000
+			targetBody = HomeWorld()
+			title = Keep the station in a stable orbit with a perigee greater than 250 km and an apogee less than 500 km
+		}
+		PARAMETER
+		{
+			name = Multi-Month Habitation Analysis
+			type = RP1CollectScience
+			title = Conduct Multi-Month Habitation Analysis of your astronauts for @/durationText days
```

Comment:
This can be quite misleading. First of all, the contract doesn't check for any number of days at all. Instead it just looks that a predefined % of the experiment has been completed. The first level appears to complete somewhere around 243 days instead. For example if you send a resupply 120 days in then you end up waiting for another 123 days for it to tick off. Maybe it would be best to mention the experiment completion % instead? Or say that the experiment has to have been running for 243 days out of 730.